### PR TITLE
LRQA-66587

### PR DESCRIPTION
**Pull request by thaisfrota**
[LRQA-66587](https://issues.liferay.com/browse/LRQA-66587)

### CODE REVIEW

**Comment by thaisfrota** [comment]
ci:test:sf

**Review by emanuelfisilva — COMMENTED** [review]
edit: please address inline requests

**Comment by emanuelfisilva** [comment]
ci:test:sf

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsStructure.testcase [hunk 1]
```diff
@@ -178,6 +178,168 @@ definition {
 			value1 = "Nome da Estrutura WC");
 	}
 
+	@description = "This test validates the creation and edition of localized fields as well as deletion of structure"
+	@priority = "5"
+	test CreateEditAndDeleteLocalizedStructure {
+		WebContentStructures.addCP(structureName = "WC Structure Title");
```
Comment (emanuelfisilva) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsStructure.testcase:184]: @thaisfrota please, add the `property portal.acceptance = "true";`

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsStructure.testcase [hunk 2]
```diff
@@ -478,6 +478,204 @@ definition {
 			fieldName = "Text");
 	}
 
+	@description = "This test validates the creation and edition of localized fields as well as deletion of structure in different instance language"
+	@priority = "5"
+	test EditAndDeleteWithDifferentInstanceLanguage {
+		ApplicationsMenu.gotoPortlet(
```
Comment (emanuelfisilva) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsStructure.testcase:484]: @thaisfrota please, add the property portal.acceptance = "true";

**Review by emanuelfisilva — CHANGES_REQUESTED** [review]
Please, see my inline requests. Thanks

**Comment by emanuelfisilva** [comment]
Thanks @thaisfrota . Made the changes and forwarded: https://github.com/emanuelfisilva/liferay-portal/pull/151

**Pull request closed without merging**